src/library/coverartutils.h

```diff
@@ -30,6 +30,8 @@ class CoverArtUtils {
     // Extracts the first cover art image embedded within the file.
     static QImage extractEmbeddedCover(
             mixxx::FileAccess trackFileAccess);
+    static QImage extractEmbeddedCover(
+            TrackPointer pTrack);
```

Comment:
whats the advantage of this overload over the `FileAccess` version?

Reply:
If we have a track, we can us it directly and bypass the track lookup logic.

Reply:
What do we gain from that?